distributions/aws/examples/rds-s3/README.md

```diff
@@ -77,12 +77,12 @@ Replace `YOUR_CLUSTER_REGION`, `YOUR_CLUSTER_NAME` and `YOUR_S3_BUCKET` with you
 python auto-rds-s3-setup.py --region YOUR_CLUSTER_REGION --cluster YOUR_CLUSTER_NAME --bucket YOUR_S3_BUCKET
 ```  
 
-##### **Advanced customization**
+### Advanced customization
 The script applies some sensible default values for the db user password, max storage, storage type, instance type etc but if you know what you are doing, you can always tweak those preferences by passing different values.  
 You can learn more about the different parameters by running `python auto-rds-s3-setup.py --help`.  
 
 
-#### **Manual setup**
+### 2.2 **Option 2: Manual Setup**
 If you prefer to manually setup each components then you can follow this manual guide.  
 1. Create S3 Bucket
```

Comment:
shouldn't this be 2.2.1 ?

Reply:
I think that gets messy. Also I checked the other READMEs and it's kinda this way. I am ok to change though if we want.